tests/cpython-tests/Makefile

```diff
@@ -2,18 +2,18 @@ TOP=$(abspath ../..)
 include $(TOP)/defs.mak
 
 APPBUILDER=$(TOP)/scripts/appbuilder
-HEAP_SIZE="4G"
-OPTS += --memory-size $(HEAP_SIZE) --nobrk
+
+OPTS = --nobrk --app-config-path config.json
```

Comment:
nit: --nobrk can also be specified in config.json

Reply:
Added "NoBrk" in config.json and keep command line options for Linux target